apps/jobbrowser/src/jobbrowser/templates/mini_job_browser.mako

```diff
@@ -0,0 +1,357 @@
+## limitations under the License.
+<%
+import sys
+
+from desktop.conf import CUSTOM, IS_K8S_ONLY
+from desktop.views import commonheader, commonfooter
+from desktop.webpack_utils import get_hue_bundles
+from jobbrowser.conf import MAX_JOB_FETCH, ENABLE_QUERY_BROWSER
+from webpack_loader.templatetags.webpack_loader import render_bundle
+
+if sys.version_info[0] > 2:
+  from django.utils.translation import gettext as _
+else:
+  from django.utils.translation import ugettext as _
+%>
+
+% if not is_embeddable:
+${ commonheader("Job Browser", "jobbrowser", user, request) | n,unicode }
+% endif
+
+<span class="notebook">
+  % if not is_embeddable:
+  <link rel="stylesheet" href="${ static('notebook/css/notebook.css') }">
+  <link rel="stylesheet" href="${ static('notebook/css/notebook-layout.css') }">
+  <style type="text/css">
+  % if CUSTOM.BANNER_TOP_HTML.get():
+    .show-assist {
+      top: 110px!important;
+    }
+    .main-content {
+      top: 112px!important;
+    }
+  % endif
+  </style>
+  % endif
+  
+  <link rel="stylesheet" href="${ static('jobbrowser/css/jobbrowser-embeddable.css') }">
+  <link rel="stylesheet" href="${ static('desktop/ext/css/bootstrap-datepicker.min.css') }">
+  <link rel="stylesheet" href="${ static('desktop/ext/css/bootstrap-timepicker.min.css') }">
+  <link rel="stylesheet" href="${ static('desktop/css/bootstrap-spinedit.css') }">
+  <link rel="stylesheet" href="${ static('desktop/css/bootstrap-slider.css') }">
+  
+  % for bundle in get_hue_bundles('miniJobBrowser'):
+    ${ render_bundle(bundle) | n,unicode }
+  % endfor
+  
+  <script src="${ static('desktop/ext/js/bootstrap-datepicker.min.js') }" type="text/javascript" charset="utf-8"></script>
+  <script src="${ static('desktop/ext/js/bootstrap-timepicker.min.js') }" type="text/javascript" charset="utf-8"></script>
+  <script src="${ static('desktop/js/bootstrap-spinedit.js') }" type="text/javascript" charset="utf-8"></script>
+  <script src="${ static('desktop/js/bootstrap-slider.js') }" type="text/javascript" charset="utf-8"></script>
+  <script src="${ static('oozie/js/dashboard-utils.js') }" type="text/javascript" charset="utf-8"></script>
+  
+  % if ENABLE_QUERY_BROWSER.get():
+  <script src="${ static('desktop/ext/js/dagre-d3-min.js') }"></script>
+  <script src="${ static('jobbrowser/js/impala_dagre.js') }"></script>
+  % endif
+  
+  <div id="jobbrowserMiniComponents" class="jobbrowser-components jobbrowser-mini jb-panel">
+  
+  % if not is_embeddable:
+    <a title="${_('Toggle Assist')}" class="pointer show-assist" data-bind="visible: !$root.isLeftPanelVisible() && $root.assistAvailable(), click: function() { $root.isLeftPanelVisible(true); }">
+      <i class="fa fa-chevron-right"></i>
+    </a>
+  % endif
+  
+    <div class="mini-jb-context-selector">
+      <!-- ko component: {
+        name: 'hue-context-selector',
+        params: {
+          connector: { id: 'impala' },
+          compute: compute,
+          ##namespace: namespace,
```

Comment:
What is this ##?

Reply:
`##` is indeed commented out code. I mainly focused on code style and the minimum required changes for the refactoring so previous commented out code was left as is.

In this particular case the comment indicates to me that this is potentially WIP and there could have been unfinished work to make this work work properly when there is more than one namespace. I guess a comment with a TODO would have been ideal when the comment was introduced. 🤷‍♂️